neomodel/relationship_manager.py

```diff
@@ -453,25 +509,44 @@ class ZeroOrMore(RelationshipManager):
     """
     A relationship of zero or more nodes (the default)
     """
-    description = "zero or more relationships"
-
-
-def _relate(cls_name, direction, rel_type, cardinality=None, model=None):
```

Comment:
Refactored RelationshipTo, RelationshipFrom, Relationship from methods to classes for consistency (Property objects are classes, not methods).
This is not a breaking change.